internal/provider/kubernetes/controller_offline.go

```diff
@@ -75,6 +75,21 @@ func NewOfflineGatewayAPIController(
 		envoyGateway:      cfg.EnvoyGateway,
 		mergeGateways:     sets.New[string](),
 		extServerPolicies: extServerPoliciesGVKs,
+		// We assume all CRDs are available in offline mode.
+		backendCRDExists:       true,
```

Comment:
set this to false will pass the test, prevent backend being added to resources tree twice

Reply:
+1 lets do this, so this gets into v1.4, and #6021 can be enhanced and added into a future patch release

Reply:
done https://github.com/envoyproxy/gateway/pull/6020/commits/7afd3b2cf7ea57926cfc38b1c018dfed15eea38a